bioconda_utils/pkg_test.py

```diff
@@ -104,6 +110,8 @@ def test_package(
     base_image : None | str
         Specify custom base image. Busybox is used in the default case.
 
+    conda_image : None | str
+        Specify conda image to install the package with.
```

Comment:
`...conda docker image to install the package with during the mulled based tests.` ?

Reply:
Yes, will update once we have a new `galaxy-lib` for this.